Services/Interfaces/web_interface/models/configuration.py

```diff
@@ -877,6 +923,10 @@ def get_full_exchange_list(remove_config_exchanges=False):
     return [exchange for exchange in full_exchange_list if constants.UPDATED_CONFIG_SEPARATOR not in exchange]
 
 
+def get_default_exchange():
+    return ccxt.async_support.binance.__name__
```

Comment:
Shouldn't we use the default profile exchange?

Reply:
We do already, this is if there is no enabled exchange in the current profile

Reply:
👍